modules/dhcp_common/dhcp_common.rb

```diff
@@ -32,4 +32,12 @@ class NotImplemented < RuntimeError; end
   def self.ipv4_to_i(ipv4_address)
     ipv4_address.split('.', 4).inject(0) { |i, octet| (i << 8) | octet.to_i }
```

Comment:
Unrelated to this patch, but isn't this equal: ```IPAddr.new(ipv4_address).to_i```?

Reply:
Probably, but I didn't want to instantiate an IPAddr object just to calculate an integer representation of an ip address. Moreover, this function is used someplace else already (it was originally added as a more efficient way to do this conversion, if I remember correctly), no reason not to use it.